### ISSUE TITLE
Approach to Ranking problems

### ISSUE THREAD

**Issue by binga** [dmlc/xgboost#352]
Hi Tianqi,

I am working on a binary classification problem where the evaluation metric is the AUC. I am using XGBoost and getting good results. However, I believe I can push it a little more by getting the right set of hyper-parameters. To my surprise, the AUC scores I'm obtaining (using CV) are better with the "binary:logistic" objective than the "rank:pairwise" objective.

Also, there's a good amount of imbalance. I have tried using scale_pos_weight but of no success. Any tips on how to approach ranking problems using XGBoost?

Thank you for the incredible library! :-)

**Comment by tqchen** [comment]
I think classification loss should work fine on your case. However, note that different objectives and weights might results in different optimal set of hyper parameters, so you want to run some tests to fined out
